docs/platform/release-notes/tizen-6-0-m2.md

```diff
@@ -0,0 +1,407 @@
+  - Platform API for UWB has been added.
+  - UWB manager daemon has been added.
+  - It has been tested with Decawave’s dwm1001 module on RPI4.
+  - Devices with UWB modules will be able to measure distance between them.
+
+
+### Security
+
+#### New and changed features
+
+- Security monitor
+  - Query-based security API has been added.
+  - Type-safe query builder has been added.
+  - Support for Device Policy Management has been added.
+- Privileges
+  - New privilege has been added.
+    - network.route
+
+
+### Service framework
+
+#### New and changed features
+
+- Location Framework
+  - Support for getting velocity accuracy has been added.
+  - Support for getting location bounds to check on edge with tolerance has been added.
+    - Support for checking on edge with tolerance for polygon has been added.
+    - Support for checking on edge with tolerance for circle has been added.
+    - Support for checking on edge with tolerance for rectangle has been added.
+- Battery-Monitor Framework
+  - Support for resetting battery dump data has been added.
+  - Support for dumping application id with event of battery dump has been added.
+- Context Framework
+  - Motion API has been deprecated.
+
+#### Fixes
+
+- Context Framework
+  - Fix for potential security vulnerability has been added.
+- Location Framework
+  - Precision improvement in getting distance between two locations has been added.
+
+
+### Web framework
+
+#### New and changed features
+
+- Web engine
+  - Feature support
+    - Support for OffscreenCanvas has been added: OffscreenCanvas enables canvas rendering apart from the main thread.
+  - Feature change/fix
+    - Strict MIME type for module scripts: Loading module script from the local file feature has been removed.
+    - App control URL bug fix: appcontrol:// scheme now able to launch application from encoded URL.
+- Web runtime
+  - Device API has been added for wrt-service.
+  - Web server and web server has been added as a wrt-service built-in module.
+  - WRT.js API management policy has been added.
+    - API has been changed from generic js to type script.
+    - API compatibility checker has been added.
+
+#### Fixes
+
+- Compatibility enhancement
+  - The problem of autofill operation in major web site has been improved.
+- Stability enhancement
+  - Unexpected random crash while clean up sequence has been fixed.
+
+
+### Lightweight Web Solution
+
+#### New and changed features
+
+- Rendering engine
+  - Data Element's DOM API has been added.
+  - Performance Web API set has been added.
+  - Range Web API set has been added.
+- JS Engine	
+  - ES10 features have been added.
+    - Array.prototype.flat
+    - Array.prototype.flatMap
+    - Object.fromEntries
+    - String.prototype.trimStart
+    - String.prototype.trimEnd
+  - Following features' syntax has been updated.
+    - Optional catch binding parameters
+    - Allowing U+2028 (LINE SEPARATOR) and U+2029 (PARAGRAPH SEPARATOR) in string literals
```

Comment:
Global: Kindly check the periods of all sub-bullets, at some places you have missed, and some places you have kept it. 
Please ensure that you keep the period at the end of all sub-bullet points as is the case with previous [release notes](https://github.com/Samsung/tizen-docs/pull/1123/files).

Reply:
I wrote them without the periods because they are just items (or features). 
In even previous release notes, general items were written without the period.
Please double check them. If i am wrong, please keep the periods.

Reply:
> I wrote them without the periods because they are just items (or features).
>In even previous release notes, general items were written without the period.
> Please double check them. If i am wrong, please keep the periods.

Agreed the features/general items are without periods, but everything else needs periods. Kindly verify and do the needful.

Reply:
@mijin85cho and @nawab-samsung  Thank you.